user/service.py

```diff
@@ -6,12 +6,20 @@
 import user.dao as user_dao
 import user_token.service as token_service
 from db_python_util.db_classes import User
+from db_python_util.db_exceptions import EntityNotFoundException
 
+import be_exceptions.validation_exceptions as validation_exceptions
 def get(user_id: str = None) -> List[User] | Optional[User]:
     """Get a user by ID, or all users if no ID is given."""
 
     if user_id is not None:
-        return user_dao.get_by_id(user_id)
+        try:
+            result =  user_dao.get_by_id(user_id)
+            if result == None:
```

Comment:
unless something has changed recently I don't think `get_by_id` should ever return `None` - it should already be doing this validation. I see there's another `except` branch to catch that error which I guess works but I have to ask why not just let that exception bubble up and catch it in the controller? (which I think is what was happening already?)